src/index.html

```diff
@@ -8,6 +8,112 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div
+        class="stars__star">
```

Comment:
wrong syntax, fix everywhere. 
```suggestion
      <div class="stars__star">
```

Reply:
done